src/JackTrip.cpp

```diff
@@ -159,9 +159,10 @@ void JackTrip::setupAudio(
 #ifdef WAIRTOMASTER // WAIR
         qDebug() << "mPeerAddress" << mPeerAddress << mPeerAddress.contains(gDOMAIN_TRIPLE);
         QString VARIABLE_AUDIO_NAME = WAIR_AUDIO_NAME; // legacy for WAIR
-        QByteArray tmp = QString(mPeerAddress).replace(":", ".").toLatin1();
-        if(mPeerAddress.toStdString()!="")
-            mJackClientName = tmp.constData();
+        //Set our Jack client name if we're a hub server or a custom name hasn't been set
+	if(mPeerAddress.toStdString()!="" && (mJackClientName == gJackDefaultClientName || mJackTripMode == SERVERPINGSERVER)) {
```

Comment:
Seems a bit excessive to have an if spread over 4 lines. Maybe this would be a compromise that still splits the two parts of the and boolean in an obvious way? (But either way works really.)

`if (mPeerAddress.toStdString() != "" && 
    (mJackClientName == gJackDefaultClientName || mJackTripMode == SERVERPINGSERVER)) {`

Reply:
Gah, it didn't preserve my line break. Let's try that again. 

if (mPeerAddress.toStdString() != "" &&
    (mJackClientName == gJackDefaultClientName || mJackTripMode == SERVERPINGSERVER)) {

(But with an appropriate indent on the 2nd line. Sorry, don't know what I'm doing with formatting on here.)

Reply:
That's also fine. My concern was the missing separation between both parts of &&.

Reply:
Makes sense. And I agree. Thanks.

Reply:
```suggestion
	  if ( mPeerAddress.toStdString() != "" &&
	      (mJackClientName == gJackDefaultClientName || mJackTripMode == SERVERPINGSERVER)) {
```